### PR TITLE
Add State

### PR DESCRIPTION
**Pull request by dracco1993**

### CODE REVIEW

**Review by NotMePipe — APPROVED** [review]

**Review by GingerJakeDaBoi — COMMENTED** [review]
VERY COOKED

**Review by GingerJakeDaBoi — APPROVED** [review]

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -309,7 +315,12 @@ public void teleopPeriodic() {
                 - RobotConstants.config.intake().k_ejectPivotAngle))) {
       m_intake.setIntakeState(IntakeState.EJECT);
       m_intaking = false;
-    } else if (m_operatorController.getWantsShoot() && m_intake.isAtStow()) {
+    } else if (getWantsEjectFinished) {
+      m_intake.setIntakeState(IntakeState.INTAKE);
+      m_intaking = true;
+    }
+    
+    else if (m_operatorController.getWantsShoot() && m_intake.isAtStow()) {
```
Comment (snydecor000) [inline src/main/java/frc/robot/Robot.java]: Unnecessary spacing change

**Review by kryllyxofficial01 — APPROVED** [review]
hunter cooked

File: src/main/deploy/pathplanner/paths/MidRing, PodiumRing.path [hunk 2]
```diff
@@ -49,7 +49,7 @@
     "rotateFast": false
   },
   "reversed": false,
-  "folder": "Center 4.5",
+  "folder": null,
```
Comment (snydecor000) [inline src/main/deploy/pathplanner/paths/MidRing, PodiumRing.path]: why

**Review by snydecor000 — COMMENTED** [review]
Very cool commits.  Fix the spacing issue :|

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -309,7 +315,12 @@ public void teleopPeriodic() {
                 - RobotConstants.config.intake().k_ejectPivotAngle))) {
       m_intake.setIntakeState(IntakeState.EJECT);
       m_intaking = false;
-    } else if (m_operatorController.getWantsShoot() && m_intake.isAtStow()) {
+    } else if (getWantsEjectFinished) {
+      m_intake.setIntakeState(IntakeState.INTAKE);
+      m_intaking = true;
+    }
+
```
Comment (dracco1993) [inline src/main/java/frc/robot/Robot.java]:
```suggestion
```

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (dracco1993) [reply src/main/java/frc/robot/Robot.java]: Done.

File: src/main/java/frc/robot/Robot.java [hunk 4]
```diff
@@ -309,7 +315,11 @@ public void teleopPeriodic() {
                 - RobotConstants.config.intake().k_ejectPivotAngle))) {
       m_intake.setIntakeState(IntakeState.EJECT);
       m_intaking = false;
-    } else if (m_operatorController.getWantsShoot() && m_intake.isAtStow()) {
+    } else if (getWantsEjectFinished) {
+      m_intake.setIntakeState(IntakeState.INTAKE);
+      m_intaking = true;
+    }
+    else if (m_operatorController.getWantsShoot() && m_intake.isAtStow()) {
```
Comment (dracco1993) [inline src/main/java/frc/robot/Robot.java]:
```suggestion
    } else if (m_operatorController.getWantsShoot() && m_intake.isAtStow()) {
```

**Pull request merged**